### DISCUSSION TITLE
How to close the floating window?

### DISCUSSION THREAD

**Discussion by resistorsoftware** [folke/trouble.nvim#608] [Q&A]
I had some LSP info in a floating window, an open buffer as a source for the window, and neo-tree open. Nothing I did could close the floating window. It was responsive to key strokes, I could click with the mouse inside and outside of it, but nothing would close this float that I tried. ESC, q, nothing obvious to me anyway... 

So what is the magic command to close the float if you're sure you're in the float, ie) see the cursor in there?

**Comment by dre-bk201** [comment]
Did you figure it out?

**Reply by resistorsoftware** [reply]
I assume that because this floating window is simply a neovim window, the *fclose* command would work, as would the *<C-w>q* but because I could not regenerate the floating window that became *Lost in Space space space...* I never did get to test these theories out. But since I now understand that floating windows close with *fclose* commands, I am assuming if I run into this again, that will get me out of the bind of the float being unkillable!

**Reply by dre-bk201** [reply]
![image](https://github.com/user-attachments/assets/73e4c612-a9f8-4962-81f5-7592c113767d)

I just looked further into it and by default it seems Trouble windows stay there when you press "return" because it executes the "jump" command but pressing "o" does "jump_close" which was the behaviour I was expecting.

**Comment by resistorsoftware** [comment]
OK. Not sure who picked lowercase 'o' to close something, but hey, why not!
